flake.nix

```diff
@@ -251,7 +251,7 @@
                   "-c"
                   ''
                     ${pkgs.ogmios}/bin/ogmios \
-                      --host 0.0.0.0 \
+                      --host ogmios \
```

Comment:
Is this going to work when making websocket requests outside of the runtime (e.g. from the browser)? I'm not sure if it will

Reply:
All I can say is that I can run Pkh2Pkh example.

I got help from @klntsky in this change.

Reply:
OK, thanks for confirming